### PR TITLE
Publish v1.0.0-rc.2

### PR DESCRIPTION
**Pull request by daxpedda**
Builds on #36.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -14,7 +14,7 @@ license = "MIT OR Apache-2.0"
 name = "derive-where"
 readme = "README.md"
 repository = "https://github.com/ModProg/derive-where"
-rust-version = "1.34.2"
+rust-version = "1.34"
```
Comment (daxpedda) [inline Cargo.toml:17]: I noticed that skipping the minor version seems to very common in other crates right now, it looks reasonable to me and is more correct anyway. 1.34.2 doesn't actually fix anything that concerns us.

**Comment by daxpedda** [comment]
Another connection issue in the CI.

File: Cargo.toml (hunk 1 above)
Comment (ModProg) [reply Cargo.toml:17]: sounds good

**Comment by daxpedda** [comment]
Re-based and good to go @ModProg.

**Review by ModProg — APPROVED** [review]

**Comment by ModProg** [comment]
only the date is in the past :smile:

**Pull request merged**